src/Symfony/Component/Translation/Command/TranslationLintCommand.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Completion\CompletionInput;
+use Symfony\Component\Console\Completion\CompletionSuggestions;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Translation\Exception\ExceptionInterface;
+use Symfony\Component\Translation\TranslatorBagInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * Lint translations files syntax and outputs encountered errors.
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+#[AsCommand(name: 'lint:translations', description: 'Lint translations files syntax and outputs encountered errors')]
+class TranslationLintCommand extends Command
+{
+    private SymfonyStyle $io;
+
+    public function __construct(
+        private TranslatorInterface&TranslatorBagInterface $translator,
+        private array $enabledLocales = [],
+    ) {
+        parent::__construct();
+    }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestOptionValuesFor('locales')) {
+            $suggestions->suggestValues($this->enabledLocales);
+        }
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputOption('locales', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'Specify the locales to lint.'),
```

Comment:
Does it make sense to use the configured locales as default?

Reply:
I've reused the same logic than the [`TranslationPullCommand`](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Translation/Command/TranslationPullCommand.php#L127), but looks like we can use `$this->enabledLocales` as default value without breaking the tests